onnxruntime/core/framework/execution_providers.h

```diff
@@ -156,5 +170,5 @@ class ExecutionProviders {
   // Whether the CPU provider was implicitly added to a session for fallback (true),
   // or whether it was explicitly added by the caller.
   bool cpu_execution_provider_was_implicitly_added_ = false;
-};
+};  // namespace onnxruntime
```

Comment:
```suggestion
};
```

Reply:
Don't know how that snuck in there. Fixed